apps-rendering/src/components/FileInput/styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+import { text } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	focusHalo,
+	neutral,
+	remHeight,
+	remSpace,
+	textSans,
+} from '@guardian/source-foundations';
+import { darkModeCss } from 'styles';
+
+export const fieldSupportingStyles = css`
+	${textSans.small()};
+	color: ${neutral[60]};
+`;
+
+export const fieldLabelStyles = css`
+	${textSans.small()};
+	color: ${neutral[7]};
+	${darkModeCss`
+		color: ${neutral[86]};
+		`}
+`;
+
+export const customUpload = (format: ArticleFormat): SerializedStyles => css`
+	${textSans.small()};
```

Comment:
This component could be a good contender for source development kitchen in the future! Then I can use it in my component too.

Reply:
Yeah this whole file input could be put into source kitchen really! Great shout!

Reply:
We could even stick the deadline component in there?